js/script.js

```diff
@@ -0,0 +1,88 @@
+//To Play Game
+const playGame = document.querySelector(".play-game");
+const homeScreen = document.querySelector(".home-screen");
+const gameSCreen = document.querySelector(".game-screen");
+const victoryScreen = document.querySelector(".victory-screen");
+
+playGame.addEventListener("click", function() {
+    homeScreen.remove();
+    gameSCreen.classList.remove("hidden");
+});
+
+// Create an array of images from folder
+const imageArray = [
+    "img/1.png",
+    "img/2.png",
+    "img/3.png",
+    "img/4.png",
+    "img/5.png",
+    "img/6.png",
+    "img/7.png",
+    "img/8.png",
+    "img/9.png",
+    "img/10.png",
+    "img/11.png",
+    "img/12.png"
+];
+
+//To select 8 unique random images
+var uniqueImages = [];
+while (uniqueImages.length < 8) {
+    random = imageArray[Math.floor(Math.random() * 12)];
+    if (uniqueImages.indexOf(random) === -1)
+        uniqueImages.push(random);
+}
+// to save the unique images in two random cards
+const gameGrid = uniqueImages
+    .concat(uniqueImages)
+    .sort(() => 0.5 - Math.random());
+
+const cardImages = document.querySelectorAll(".card-image");
+gameGrid.forEach((imgStr, index)=> {
+    cardImages[index].setAttribute("src", imgStr);
+})
+
+// flip card and display images
+const cards = document.querySelectorAll(".card");
+for (let i = 0; i < cards.length; i++) {
+    cards[i].addEventListener("click", flipCard)  
+}
+
+function flipCard() {
+    this.classList.toggle("flip");
+    //starts time after first click on card
+    if (timeStart === false) {
+        timeStart = true;
+        timer();
+    }
+    //moves count after each click on card
+    movesCounter();
+}
+// initialize time count
+const timeCount = document.querySelector(".time-read");
+let minutes = 0;
+let seconds = 0;
+let timeStart = false;
+
+function timer() {
+    let time = setInterval(function() {
+        seconds++;
+        if (seconds < 10) {
+            seconds = `0${seconds}`;
+        }
+        if (seconds >= 60) {
+            minutes++;
+            seconds = "00";
+        }
+        timeCount.innerHTML = minutes + " : " + seconds;
+    }, 1000);
+}
+
+// initialize moves count  
+const movesCount = document.querySelector(".count-value");
+let moves = 0;
+
+function movesCounter() {
+    moves++;
+    movesCount.innerHTML = moves;
```

Comment:
`innerText` is better in this case since the content is not HTML but a simple text value. When you use `innerHTML` the browser tries to render the string you pass it which takes a little time. It might be insignificant in this case, but in a large project it adds up.

Do this same for other parts of the code that uses `innerHTML` where the content is not actually HTML.